packages/attention/stories/Attention.stories.tsx

```diff
@@ -41,6 +41,44 @@ export function CalloutResettingRoleAndAriaLabel() {
   )
 }
 
+export function Highlight() {
+  const [show, setShow] = React.useState(false)
+  const targetEl = React.useRef()
+
+  return (
+    <div className='flex flex-col justify-between h-[200]'>
+      <Button
+        small
+        aria-expanded={show}
+        aria-controls='highlight-attention-example'
+        utility
+        onClick={() => setShow(!show)}
+        className='w-max mb-0'
+      >
+        Show an onboarding hint
+      </Button>
+      <div>
+        <div ref={targetEl} className="w-2/3">
+          <Box info>
+            <h1>I am a box full of info</h1>
+          </Box>
+        </div>
+        <Attention 
+          highlight
+          placement='top'
+          isShowing={show}
+          targetEl={targetEl}
+          id='highlight-attention-example'
```

Comment:
I think the id needs to be added to the component.tsx file in the wrapperClasses-div. Otherwise I cannot find this id on the Attention-component when inspecting the DOM. :)

Reply:
Ah, good catch! Fixed in [78ed95a](https://github.com/warp-ds/react/pull/145/commits/78ed95a9950706aabedb14e2a433c436e30c5d9f)